EosioSwift/EosioRpcProvider/Models/EosioInt64.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  EosioInt64.swift
+//  EosioSwift
+
+//  Created by Steve McCoole on 8/15/19
+//  Copyright (c) 2017-2019 block.one and its contributors. All rights reserved.
+//
+
+import Foundation
+
+/// Meta type that can hold a 64 bit signed value that can come from the server as a Int64 or a String.
+public enum EosioInt64: Codable {
+    /// Value as a `Int64`.
+    case int64(Int64)
+    /// Value as a `String`.
+    case string(String)
+
+    public var value: Int64 {
+        switch self {
+        case .int64(let value):
+            return value
+        case .string(let str):
+            let val: Int64? = Int64(str)
+            return val ?? 0
+        }
+    }
+
+    /// Initialize from a decoder, attempting to decode as a `Int64` first. If that is unsuccessful, atttempt to decode as `String`.
```

Comment:
`atttempt` -> `attempt`

Reply:
(If this exists in the `EosioUInt64.swift` file too, could you correct that also?

Reply:
Fixed